services/src/main/java/org/keycloak/forms/login/freemarker/FreeMarkerLoginFormsProvider.java

```diff
@@ -168,6 +168,7 @@ public Response createResponse(UserModel.RequiredAction action) {
                 page = LoginFormsPages.LOGIN_UPDATE_PASSWORD;
                 break;
             case VERIFY_EMAIL:
+                this.attributes.put("verifiedEmail",user.getEmail());
```

Comment:
One small note: When looking at the sources, I wonder that instead of `verifiedEmail`, it may be slightly better to use something like:
```
UpdateProfileContext userBasedContext = new UserUpdateProfileContext(realm, user);
attributes.put("user", new ProfileBean(userCtx, formData));
```
And then in the FTL template, you can use maybe something like:
```
 <p class="instruction">${msg("emailVerifyInstruction1", user.email)}</p> 
```

The reason why proposing this is, that this might be useful in case that administrators want to use other informations of user in their FTL template. For example something like: "Hello John Doe, the message was sent to your email johndoe@gmail.com"  .

In other words, administrators won't be restricted to use only email as in theory, they may want to use other attributes of the user.